packages/nextjs/src/api/index.ts

```diff
@@ -75,6 +76,21 @@ export const getTenants = async (
   return parseHttpResponse(res);
 };
 
+/**
+ *
+ * @param headers
+ */
+export const getMeAuthorization = async (
```

Comment:
shouldn't it replace the getUsers(getMe) request?

Reply:
no its 2 different requests..